libcxx/utils/libcxx/header_information.py

```diff
@@ -170,7 +170,7 @@ def __hash__(self) -> int:
     "linalg",
     "rcu",
     "spanstream",
-    "stacktrace",
+    "stacktrace",  # TODO(stacktrace23): remove this when stacktrace is taken out of experimental
```

Comment:
You should probably just declare this header as implemented even if marked as "experimental" but I am not too sure about it.